### PR TITLE
Updated images and links 

### PR DESCRIPTION
**Pull request by DannyRuchtie**
## What kind of change does this PR introduce?
- [ ] Updated images and added links
- [ ] Redirect are done on the server-side by Bogdan 

## What is the current behavior?
- Old images
- Links not working

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5615/commits/9d8b08269c2ba0a2a542e77d82465bd565d9fa61 is at https://pr5615.build.csb.dev/s/new.

File: packages/homepage/src/pages/podcasts/codesandbox-podcast.js [hunk 1]
```diff
@@ -95,35 +108,34 @@ const VersionOne = ({
...
-            image {
-              publicURL
-            }
-          }
-        }
-      }
-    }
-  }
-`;
+// export const query = graphql`
```
Comment (SaraVieira) [inline packages/homepage/src/pages/podcasts/codesandbox-podcast.js:118]: Why did you remove this? You gonna have to put it back when there are episodes

Comment (SaraVieira) [reply packages/homepage/src/pages/podcasts/codesandbox-podcast.js:118]: Let me know if you want to have a call

Comment (DannyRuchtie) [reply packages/homepage/src/pages/podcasts/codesandbox-podcast.js:118]: Hi Sara, 
Yeah, I had an issue when I deleted the first episode's MD  files. All worked well locally, but it would not build the PR. 
Scott could debug it and discovered that something went wrong because I deleted the MD files. 
That's why I have tried to turn it off. 

The goal is to have the pages available with the trailers but no episodes yet (next week). 
And as soon as the episodes are released to show the episodes. 

I will uncomment the parts as soon as the first episode is up. #hackhack
